src/domain/engine/Game/game.test.ts

```diff
@@ -1,3 +1,4 @@
+import 'reflect-metadata';
```

Comment:
You don't need this import. You have to delete it in order to optimize performances.